packages/web-app-skeleton/src/search/github/PreviewComponent.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <oc-button
+    type="a"
+    justify-content="left"
+    class="files-search-preview oc-flex oc-width-1-1"
+    appearance="raw"
+    target="_blank"
+    :href="repository.html_url"
+  >
+    {{ repository.name }}
+  </oc-button>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import { Repository } from './client'
+import { PropType } from 'vue'
+
+export default defineComponent({
+  props: {
+    searchResult: {
+      type: Object as PropType<Repository>,
+      default: function () {
+        return {}
+      }
```

Comment:
i think just `default: {}` should be fine

Reply:
vue/require-valid-default-prop, https://eslint.vuejs.org/rules/require-valid-default-prop.html.

Must be of type function